medic/medic-appium.js

```diff
@@ -197,9 +214,10 @@ function parseArgs() {
     return options;
 }
 
-function getLocalCLI() {
-    if (util.isWindows()) {
-        return "cordova.bat";
+function getLocalCLI(appPath) {
+    if (util.isWindows() || !fs.existsSync(path.join(getFullAppPath(appPath), "./cordova"))) {
```

Comment:
Why we fall back to default on Windows? I can see `cordova.bat` in mobilespec directory on Windows